woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2651,8 +2652,9 @@ def test_maintain_column_order_disordered_schema(sample_df):
     column_order = list(sample_df.columns)
 
     scramble_df = sample_df.ww.copy()
-    id_col = scramble_df.ww.columns.pop("id")
-    scramble_df.ww.columns["id"] = id_col
+    table_schema = scramble_df.ww.columns
+    id_col = table_schema.pop("id")
+    table_schema["id"] = id_col
```

Comment:
Worked with @thehomebrewnerd on this. Previously, calling `scramble_df.ww` after popping `id` would cause the `ColumnsNotPresentInSchemaError` to appear. We instead reference it and have verified the test continues to test what we expect here.